app/models/grda_warehouse/remote_config.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module GrdaWarehouse
+  class RemoteConfig < GrdaWarehouseBase
+    attr_encrypted :password, key: ENV['ENCRYPTION_KEY'][0..31]
+    # Be nice to S3
+    alias_attribute :s3_access_key_id, :username
+    alias_attribute :s3_secret_access_key, :password
+
+    # The following should be implemented in the concrete classes
```

Comment:
Wondering if maybe we should leave these out -- there are certainly a set of common interfaces for services, but they will differ by service class... In the future, we could use RBS interfaces.